metadata_service/api/column.py

```diff
@@ -89,3 +90,33 @@ def get(self, table_uri: str, column_name: str) -> Union[tuple, int, None]:
 
         except Exception:
             return {'message': 'Internal server error!'}, HTTPStatus.INTERNAL_SERVER_ERROR
+
+
+class ColumnBadgeAPI(Resource):
+    def __init__(self) -> None:
+        self.client = get_proxy_client()
+        self.parser = reqparse.RequestParser()
+        self.parser.add_argument('category', type=str, required=True)
+        super(ColumnBadgeAPI, self).__init__()
+
+        self._badge_common = BadgeCommon(client=self.client)
+
+    @swag_from('swagger_doc/column/badge_put.yml')
```

Comment:
the swagger doesn't look correct. any reason we have another badge put instead of reusing `swagger_doc/badge/badge_put.yml`

Reply:
cc @allisonsuarez  I think the column badge should extend the same swagger file shared by table/dashboard badge (https://github.com/amundsen-io/amundsenmetadatalibrary/blob/master/metadata_service/api/swagger_doc/badge/badge_put.yml)

Reply:
The `column` badges require another parameter (`column_name`) and it uses `table_uri` and not `id`.

Reply:
hey @alldoami if you take a look at https://github.com/amundsen-io/amundsenmetadatalibrary/blob/master/metadata_service/api/dashboard.py#L92 or https://github.com/amundsen-io/amundsenmetadatalibrary/blob/master/metadata_service/api/table.py#L199 which both takes an id as input param which the id could be a table_uri, dashboard_uri or column_uri which could unique identifies the entity resource.  With that, we won't need table_uri, column_name.

I think we should do the same here.

Reply:
@dorianj had stronger feelings about this 😅

Reply:
We also talked about this through slack, we'll need the `column_name` as it'd be more straightforward to have the API resource be `table/<table_uri>/column/<column_name>/badge/<badge_name>/<category>` instead of `table/<table_id with column/column/badge...`. We need a unique identifier to tell us which class to use (which is why we include the `column` in the API call. If we went with `id`, it would look exactly like the table badge API and we'd need to include something like the word `column` in there. I think my way is the most straightforward still.

Reply:
We also talked about this through slack, we'll need the `column_name` as it'd be more straightforward to have the API resource be `table/<table_uri>/column/<column_name>/badge/<badge_name>/<category>` instead of `table/<table_id with column/column/badge...`. We need a unique identifier to tell us which class to use (which is why we include the `column` in the API call. If we went with `id`, it would look exactly like the table badge API and we'd need to include something like the word `column` in there. I think my way is the most straightforward still.

Reply:
any reason the API can't be `/column/<path:id>/badge/<badge>`

Reply:
Also if we change it to using `swagger_doc/badge/badge_put.yml`, then the `id`s will be accepting different formatted values for table and column, which could also cause confusion. It's an easy fix it you want me to change it, but want to explain why we implemented this way before changing.

Reply:
I see, thanks for sharing the context, I am ok with your format to make it consistent with the other column API.